src/ui/components/SecondaryToolbox/ReactDevTools.tsx

```diff
@@ -123,6 +124,20 @@ class ReplayWall implements Wall {
     this._listener?.({ event: "stopInspectingNative", payload: true });
   }
 
+  // This is called  by the `useEffect` in the React DevTools panel.
+  async setUpRDTInternalsForCurrentPause() {
+    // Preload the React DevTools Backend for this pause
+    await this.ensureReactDevtoolsBackendLoaded();
+    // We should also kick off a request to pre-fetch the `Map<nodeId, fiberId>` lookup for this pause.
+    // That way, we have the data on hand when the user clicks on a node.
+    // This is async, but we won't wait for it here.
+    this.fetchNodeIdsToFiberIdsForPause();
```

Comment:
I think I know why this is desirable in terms of speeding up the RDT node picker, but this will fire every time the execution point changes when the RDT component is mounted/visible, which means that in the most common case (where the node picker isn't even used) we'll be doing _a lot_ of "unnecessary" work.

What's the expected impact/overhead of that? Is it worth it?

After the performance improvement, do things feel okay enough without this?

Reply:
I feel good about this, yeah. We're already doing one eval per pause anyway while the RDT is visible to inject the backend logic, this adds a second to pre-fetch the fiber IDs. In my tests against one of our own FE client recordings, that's taking ~150ms.  We do lots of evals all the time anyway, so I don't think this is meaningfully expensive.

Reply:
What about the slow path though? That's a lot of work, no?

Reply:
I think it would be better to not do it until/unless the user is likely to be interacting with the node picker. Do we have a way to anticipate that? Maybe only do it if the React panel is focused?

Reply:
We do also use it for the hover behavior now as well, which can happen without any immediate focus.  

If we make the assumption that users are likely to be interacting with recent Chromium recordings (which will be more true over time), this seems acceptable.  (Especially since we know Pinterest is on current Chromium.)

Reply:
Assuming happy path, I don't think 150ms is too much to pay for an initialization cost TBH. I think that would be better than always doing this.

Reply:
> I think it would be better to not do it until/unless the user is likely to be interacting with the node picker

> I checked that same recording, and the actual eval to collect the fiber/node IDs came back in 58ms:

Especially if it's this fast.